tests/test_sqs/test_sqs.py

```diff
@@ -311,20 +311,15 @@ def test_get_queue_url_error_not_exists():
 @mock_sqs
 def test_get_nonexistent_queue():
     sqs = boto3.resource("sqs", region_name="us-east-1")
-    with pytest.raises(ClientError) as err:
```

Comment:
This particular check wasl already part of the previous test, so there's no need to replicate it